util/containerutil/settings_test.go

```diff
@@ -328,7 +328,7 @@ func TestBuildArgMatrixValidationFailures(t *testing.T) {
 	}
 }
 
-func TestParseAndvalidateURLFailures(t *testing.T) {
+func TestParseAndValidateURLFailures(t *testing.T) {
```

Comment:
This change isn't necessary, strictly speaking. But given the style here, there didn't seem to be a good reason not to make it.